.circleci/config.yml

```diff
@@ -0,0 +1,29 @@
+# 
+version: 2
+jobs:
+  build:
+    working_directory: /build
+    docker:
+      - image: docker:17.06.0-ce-git
+    steps:
+      - checkout
+      - setup_remote_docker:
+          reusable: true
+          exclusive: true
+      - run:
+          name: Install bash
+          command: apk add --no-cache bash
+      - run:
+          name: Build image
+          command: bin/build.sh
+      - deploy:
+          name: Deploy image to Docker Hub
+          branches:
+            only:
+              - feature/circleci
```

Comment:
Seems `master` branch is needed as well.
Also will this work when creating a git tag (GitHub release)?